docs/setup/install.md

```diff
@@ -132,15 +132,14 @@ cd jobset
 IMAGE_REGISTRY=<registry>/<project> make image-push deploy
 ```
 
-<!-- Uncomment once we have a prometheus setup -->
-<!-- ### Add metrics scraping for prometheus-operator -->
+### Add metrics scraping for prometheus-operator
 
-<!-- To allow [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator) -->
-<!-- to scrape metrics from jobset components, run the following command: -->
+To allow [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator)
+to scrape metrics from jobset components, run the following command:
 
-<!-- ```shell -->
-<!-- make prometheus -->
-<!-- ``` -->
+```shell
+make prometheus
+```
```

Comment:
nothing applicable to the development version?

Reply:
can you clarify what you mean?

Reply:
we have instructions to deploy jobset from the latest pre-built development version above: "Install the latest development version"; how would one install prometheus if they want to go that route?

Reply:
I see. Updated this section accordingly.